### ISSUE TITLE
OneDrive folder

### ISSUE THREAD

**Issue by Razzy76** [ChrisRfr/Win10XPE#40]
Hi, this is probably not related to your excellent project.  Is there a way I could copy OneDrive folder from an unbootable drive?  I realize I could just login OneDrive to see those files but I would still like to recover them.

**Comment by Razzy76** [comment]
Yes I do realize that (as i said in first post) but what if the OneDrive was not signed in for a while because someone forgot the password or that they don't know what they're doing.

**Comment by Razzy76** [comment]
Yep I run an IT business (although I handle more consumer than business side) as well and have the same experience as you.   I run into countless customers who had no idea what is OneDrive and have no idea what their password (but they know the PIN number).  Sometimes I am able to boot it up to windows and sign in with PIN and good to go but there are times OneDrive app was complaining about not being signed in and it's a pain to deal with customers on retrieving/resetting the password.

Thank you.

**Comment by PCAssistSoftware** [comment]
> .... customers who had no idea what is OneDrive and have no idea what their password (but they know the PIN number) ...

I'm glad it isn't just me who faces that then, I know exactly what you mean :)

**Comment by kalaklanar** [comment]
You may be able to use one of the cloning utilities to clone to another computer and the login state might be saved as well. This would have you logged in at a working computer. You can then pull the files down on that computer
